### PR TITLE
Update function progressUpdatedHandler

### LINKED ISSUE #9: Display percentage of data uploaded
Hello @wparad,

I use your project to upload big files (100 mo and more) to S3 bucket.
During the upload step the percentage of data uploaded is initially fixed at 0% and when upload is finished it's set to 100%.

In the progressUpdatedHandler function in uploadModal.vue file I use two properties of the evt (evt.loaded and evt.total) to calculate the percentage of data uploaded.

Thus, the percentage value is updated dynamically and helps to follow the progress of the upload for big files.
What do you think about this proposal ?

```javascript
const progressUpdatedHandler = evt => {
  DEBUG.log('File:', file, 'Part:', evt.part, evt.loaded, 'of', evt.total);
  // Use evt.loaded and evt.total to calculate the percentage of data uploaded
  state.completionPercentageMap[fileIndex] = Math.round((evt.loaded * 100) / evt.total);
};
```

### PR DESCRIPTION
**Pull request by Manouchka94**
Use evt.loaded and evt.total to calculate the percentage of data uploaded
resolves #9

### CODE REVIEW

**Comment by Manouchka94** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/components/uploadModal.vue [hunk 1]
```diff
@@ -147,7 +147,10 @@ const uploadFiles = () => {
 
     const progressUpdatedHandler = evt => {
       DEBUG.log('File:', file, 'Part:', evt.part, evt.loaded, 'of', evt.total);
-      state.completionPercentageMap[fileIndex] = 0;
+      if (typeof evt.loaded == "number" && typeof evt.total == "number"){
+        // Use evt.loaded and evt.total to calculate the percentage of data uploaded
+        state.completionPercentageMap[fileIndex] = Math.round((evt.loaded * 100) / evt.total);
```
Comment (wparad) [inline src/components/uploadModal.vue:152]:
```suggestion
        state.completionPercentageMap[fileIndex] = Math.round((evt.loaded * 100) / evt.total) || 0;
```

File: src/components/uploadModal.vue [hunk 2]
```diff
@@ -147,7 +147,10 @@ const uploadFiles = () => {
 
     const progressUpdatedHandler = evt => {
       DEBUG.log('File:', file, 'Part:', evt.part, evt.loaded, 'of', evt.total);
-      state.completionPercentageMap[fileIndex] = 0;
+      if (typeof evt.loaded == "number" && typeof evt.total == "number"){
+        // Use evt.loaded and evt.total to calculate the percentage of data uploaded
+        state.completionPercentageMap[fileIndex] = Math.round(evt.loaded * 100 / evt.total) || 0;
+      }
```
Comment (wparad) [inline src/components/uploadModal.vue:153]:
```suggestion
      state.completionPercentageMap[fileIndex] = Number(evt.total) ? Math.round(Number(evt.loaded) / Number(evt.total) * 100) || 0;
```

File: src/components/uploadModal.vue (hunk 1 above)
Comment (wparad) [reply src/components/uploadModal.vue]: We need to handle for NaN issues with 0.

File: src/components/uploadModal.vue (hunk 2 above)
Comment (wparad) [reply src/components/uploadModal.vue]: We should always set this value.

File: src/components/uploadModal.vue [hunk 3]
```diff
@@ -147,7 +147,7 @@ const uploadFiles = () => {
 
     const progressUpdatedHandler = evt => {
       DEBUG.log('File:', file, 'Part:', evt.part, evt.loaded, 'of', evt.total);
-      state.completionPercentageMap[fileIndex] = 0;
+      state.completionPercentageMap[fileIndex] = Number(evt.total) ? Math.round(Number(evt.loaded) / Number(evt.total) * 100) || 0;
```
Comment (wparad) [inline src/components/uploadModal.vue:150]:
```suggestion
      state.completionPercentageMap[fileIndex] = Number(evt.total) && Math.round(Number(evt.loaded) / Number(evt.total) * 100) || 0;
```

**Comment by wparad** [comment]
@Manouchka94 I've gone and merged this, thanks for this PR.

**Pull request merged**